### ISSUE TITLE
Joint optimization of pose with other method such as Voxurf

### ISSUE THREAD

**Issue by Hugo-spyder** [ActiveVisionLab/porf#2]
Hello,  great work!!! I notice you get the high quality mesh via training the Voxurf model with the refined pose. I just want to know can this method directly integrate with other method like Voxurf , Neuralangelo etc...

**Comment by JiawangBian** [comment]
Of course. We tried PoRF with different models in nerfstudio, which improves performance for all models. We believe that it can be integrated into other methods directly.

**Comment by pablovela5620** [comment]
> Of course. We tried PoRF with different models in nerfstudio, which improves performance for all models. We believe that it can be integrated into other methods directly.

Any plans on making a PR for nerfstudio? Does this also apply to gaussian splitting models like splatfacto or only nerf/sdf based representations?

**Comment by xiemeilong** [comment]
@JiawangBian Same question here, Does this also apply to gaussian splitting models like splatfacto ?
